### ISSUE TITLE
Fails on server side render: "document is not defined"

### ISSUE THREAD

**Issue by Ornhoj** [vimeo/player.js#214]
```
// Find an element by ID
if (typeof element === 'string') {
   element = document.getElementById(element);
}
```

Using `document` methods without checking is `document` is defined will cause and error when serverside rendering and importing the player.

**Suggested solution(s)**
Either test if `document` is defined before referencing it, or avoid declaring the class before `document` is defined.

**Comment by luwes** [comment]
@Ornhoj I thought this was fixed by https://github.com/vimeo/player.js/pull/150
How can this be reproduced?

**Comment by luwes** [comment]
Closed by #215
